app/controllers/case_contacts_controller.rb

```diff
@@ -10,19 +10,30 @@ def index
   # GET /case_contacts/1
   # GET /case_contacts/1.json
   def show
+    @case_contact_number = CasaCase.find(@case_contact.casa_case_id).case_number
   end
 
   # GET /case_contacts/new
   def new
     @case_contact = CaseContact.new
+    @casa_cases = current_user.casa_cases
   end
 
   # GET /case_contacts/1/edit
   def edit
+    @casa_cases = current_user.casa_cases
   end
 
   def create
-    @case_contact = CaseContact.new(case_contact_params)
+    info =
+      {
+        creator_id: current_user.id,
+        casa_case_id: params[:casa_case_id],
+        contact_type: params[:contact_type],
+        duration_minutes: params[:duration_minutes],
+        occurred_at: params[:case_contact][:occurred_at]
+      }
```

Comment:
To construct the arguments for our new `CaseContact`, can we use the `case_contact_params` (which is an instance of `CaseContactParameters`)? I'm curious if there was a reason another approach was used here. I'm also happy to pair on it, if that seems better! 😄

Reply:
I spent a long time trying to use case_contact_params last night. The method in the values/ folder changed the params so much that when the new CaseContact was created as an empty object.